appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
+
+function Rebase-Uri($originalURI)
+{
+  $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString($originalURI.IndexOf("c:/")))
```

Comment:
>IndexOf [](start = 80, length = 7)

$originalURI.IndexOf("c:/") is kind of strange for a few reasons. First, assuming the string is always a path, won't this always evaluate to zero when it succeeds? So I don't understand how it would affect the Substring() call. Second, are we sure the only drive we need to be concerned about is the C:\ drive? And if so, is the drive letter always going to be lower cased?